docs/rules/expiring-todo-comments.md

```diff
@@ -0,0 +1,56 @@
+# Add conditions to TODO comments to make them error
+
+Makes possible to pass arguments to TODO and FIXME comments to trigger errors.
+
+Currently supporting:
+
+* `[YYYY-MM-DD]` to define a due date.
+* `[> 1]` or `[>= 2]` to expire at some version (from package.json).
+* `[+package]` or `[-package]` to expire when you add/remove a package.
+* `[package > 1]` or `[package >= 2]` to expire when a package hits some version.
```

Comment:
This should use the npm install syntax for versions.
```suggestion
* `[package@>1]` or `[package@>=2]` to expire when a package hits some version.
```

Reply:
Fixed that one now

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/302/commits/024c8657e923a30832cc9c82fc87628ec9952fde